installer/seldon-core/values.yaml

```diff
@@ -0,0 +1,3 @@
+istio:
+  enabled: true
+  gateway: seldon-system/seldon-gateway
```

Comment:
by the way, our installer gateway code will also create a gateway in the seldon-system namespace that will override this, unless we improve the installer to accept reusing a gateway in another namespace

Reply:
No, this does not create gateway itself. It only points to the right gateway which (according to official instructions) user must create manually.

Reply:
Oh, I see, in that case, it makes sense.

Reply:
in the end, it's gonna be in fuseml-workloads namespace